### PR TITLE
Create Application Context

### LINKED ISSUE #100: Create Application Context
Use Application Context to set and get global variables.

### PR DESCRIPTION
**Pull request by celinecamacho**
Closes #100 

Use the new PoPApplication class to set and get global variables. This way we can pass Person and LAO information between activities.

### CODE REVIEW

**Review by jbmichel14 — APPROVED** [review]

File: app/src/main/java/com/github/dedis/student20_pop/MainActivity.java [hunk 1]
```diff
@@ -66,17 +66,19 @@ public void onClick(View view) {
                 break;
             case R.id.button_launch:
                 String name = ((EditText) findViewById(R.id.entry_box_launch)).getText().toString();
-                // For later: send LAO and organizer information
                 Person organizer = new Person("name");
```
Comment (GabrielFleischer) [inline app/src/main/java/com/github/dedis/student20_pop/MainActivity.java:69]: `organizer =  ((PoPApplication) getApplication()).getPerson()`
We should extract organizer as us here.

**Review by GabrielFleischer — APPROVED** [review]
It is ok, just one change but it comes with all the changes needed now that there is a context

File: app/src/main/java/com/github/dedis/student20_pop/MainActivity.java (hunk 1 above)
Comment (celinecamacho) [reply app/src/main/java/com/github/dedis/student20_pop/MainActivity.java:69]: Indeed, good catch

File: app/src/androidTest/java/com/github/dedis/student20_pop/MainActivityTest.java [hunk 2]
```diff
@@ -47,6 +53,8 @@ public void onClickLaunchTest() {
     @Test
     public void onClickLaunchLAOTest() {
         onView(withId(R.id.tab_launch)).perform(click());
+        onView(withId(R.id.entry_box_launch)).perform(typeText("LAO"));
+        onView(withId(R.id.entry_box_launch)).perform(closeSoftKeyboard());
```
Comment (AlexisTabin) [inline app/src/androidTest/java/com/github/dedis/student20_pop/MainActivityTest.java:57]: you can write this on one line : `.perform(typeText("LAO"), closeSoftKeyboard());`

**Review by AlexisTabin — APPROVED** [review]

File: app/src/androidTest/java/com/github/dedis/student20_pop/MainActivityTest.java (hunk 2 above)
Comment (celinecamacho) [reply app/src/androidTest/java/com/github/dedis/student20_pop/MainActivityTest.java:57]: Good tip, thanks

**Pull request merged**